OpenOversight/tests/test_routes.py

```diff
@@ -1307,3 +1310,26 @@ def test_user_can_change_dept_pref(mockdata, client, session):
 
         user = User.query.filter_by(email='jen@example.org').one()
         assert user.dept_pref == test_department_id
+
+
+def test_admin_can_update_users(mockdata, client, session):
+    with current_app.test_request_context():
+        login_admin(client)
+
+        user = User.query.except_(User.query.filter_by(is_administrator=True)).first()
+        user_id = user.id
+
+        form = EditUserForm(
+            ac_department_id=None,
```

Comment:
Heyyy so I looked at this failing test - the form validation error here (also I like that you are also living dat #pdblyfe) is:

```
{'is_area_coordinator': ['Area coordinators must have a department']}
```

Also the field on the form is `ac_department` - when I set a department here and use the field `ac_department` instead of `ac_department_id` the test passes for me. i.e.:

```
        department = random.choice(dept_choices())

        form = EditUserForm(
            ac_department=department.id,
            is_administrator='y')
```

(unless you were intentionally trying to test the behavior when a user is demoted from the role of AC on a department?)